apm-sniffer/config/agent.config

```diff
@@ -103,6 +103,7 @@ plugin.mount=${SW_MOUNT_FOLDERS:plugins,activations}
 
 # Kafka producer configuration
 # plugin.kafka.bootstrap_servers=${SW_KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+# plugin.kafka.get_topic_timeout=${SW_GET_TOPIC_TIMEOUT:5}
```

Comment:
This is not a widely used configuration, please remove it from the default agent.config.

Reply:
done.